### ISSUE TITLE
Create separate guidelines

### ISSUE THREAD

**Issue by arlin** [arlin/hip_hack_howto#24]

**Comment by arlin** [comment]
Finished this except for the follow-up  bit.  Rutger is reviewing.

**Comment by rvosa** [comment]
I am to review this document, correct? https://www.dropbox.com/s/l2js0vz0lqlutxh/guidance_docs.docx?dl=0

**Comment by arlin** [comment]
Yes, that would be great!

Also, you looked over the manuscript today.  Did you have any thoughts about how to focus this on a few key conclusions?

Arlin

On Oct 13, 2015, at 12:00 PM, Rutger Vos <notifications@github.com<mailto:notifications@github.com>> wrote:

I am to review this document, correct? https://www.dropbox.com/s/l2js0vz0lqlutxh/guidance_docs.docx?dl=0

—
Reply to this email directly or view it on GitHubhttps://github.com/arlin/hip_hack_howto/issues/24#issuecomment-147759745.

---

Arlin Stoltzfus (arlin@umd.edumailto:arlin@umd.edu)
Research Biologist, NIST; Fellow, IBBR; Adj. Assoc. Prof., UMCP
IBBR, 9600 Gudelsky Drive, Rockville, MD, 20850
tel: 240 314 6208; web: www.molevol.orghttp://www.molevol.org

**Comment by rvosa** [comment]
I am not all the way through the manuscript yet so I am not ready to say at this point, but I'll work on it.

**Comment by rvosa** [comment]
I have reviewed the document in the dropbox - but since I only have the direct link to the docx itself I don't know in what folder it needs to go. Can I get a link to a folder? Or should I email the edited version to someone?

**Comment by arlin** [comment]
I added you to the HIP folder, which has a HackathonLessonsLearned subfolder.

**Comment by rvosa** [comment]
I have reviewed and edited this document. As far as I am concerned I think it is in good shape, i.e. I would not be embarrassed if we added this is supplementary materials. Seeing that I am having some dropbox issues I cannot currently add the edited version back to the shared folder, so I have emailed my edits to @arlin - who would still have to accept the tracked changes (they are basically all trivial edits) and save the doc in the shared folder. As for this issue assigned to me: I think we can consider it done so I'll close it.
